### DISCUSSION TITLE
Tansform from a game app to an educational app for Chinese players

### DISCUSSION THREAD

**Discussion by calcitem** [calcitem/Sanmill#321] [Ideas]
According to the requirements of the relevant policies of the General Administration of Publishing in Mainland China in the past year, mobile games, whether it is an online version or a stand-alone version, must obtain a version number. After connecting to the anti-addiction system, they can be published on the Chinese app store. The premise of obtaining a version number It is necessary to first register a company, obtain a business license, and apply for a version number. The overall cost is high.

Not realistic for a non-profit open source project like this one. Therefore, in order to be listed on the app store in the mainland, it is necessary to transform the App into an educational application, that is, the main content of the App is to teach people how to play mill, introduce skills, provide exercises, etc., while the human-machine Battle as an accessory feature and hide it to the depths.

And, if it is still a game application, it needs to implement the anti-addiction function, "only provide game services to minors from 20:00 to 21:00 every day on Friday, Saturday, Sunday and legal holidays, and the rest of the time needs to be banned from games" . Accessing the anti-addiction SDK will destroy the purity of the app, which is also unrealistic.

**Comment by zicjin** [comment]
无盈利的开源项目为何一定要在大陆地区应用商店上架？

**Reply by calcitem** [reply]
感谢关注!这是为了吸引更多开发者参与这个项目!
